src/common/StreamBlobClient.js

```diff
@@ -0,0 +1,71 @@
+/* globals define */
+define([
+    'client/logger',
+    'deepforge/gmeConfig',
+    'blob/BlobClient',
+    'deepforge/StorageHelpers'
+], function(
+    Logger,
+    gmeConfig,
+    BlobClient,
+    StorageHelpers
+) {
+    const StreamBlobClient = function (logger) {
+        const params = this.getBlobClientParams(logger);
```

Comment:
As this is subclassing `BlobClient` and the hope is to contribute this back to webgme-engine, this should not modify the method signature unnecessarily. The creation of the blob client parameters should be handled by the storage adapter itself.